include/groonga/groonga.h

```diff
@@ -1170,31 +1170,46 @@ grn_obj_close(grn_ctx *ctx, grn_obj *obj);
  */
 GRN_API grn_rc
 grn_obj_reinit(grn_ctx *ctx, grn_obj *obj, grn_id domain, uint8_t flags);
-/* On non reference count mode (default):
- * This closes the following objects immediately:
+/**
+ * \brief Unlink an object.
+ *
+ * This calls \ref grn_obj_close only when the specified object (`obj`)
+ * can be closed. See the following description for details.
  *
- *   * Acceessor
- *   * Bulk
- *   * DB
- *   * Temporary column
- *   * Temporary table
+ * **Reference Count Mode**:
+ * Reference count mode manages object lifetimes by keeping track of
+ * active references. It can be enabled in two ways:
+ * - **Environment Variable**: Set `GRN_ENABLE_REFERENCE_COUNT` to
+ *   `yes` before running the program.
+ * - **API Function**: Use \ref grn_set_reference_count_enable to
```

Comment:
```suggestion
 * - **API**: Use \ref grn_set_reference_count_enable to
```

or just `**Function**`.

Reply:
fix: [5c1545b](https://github.com/groonga/groonga/pull/1955/commits/5c1545b1cc4c579bc6d4d21f7b8017e9024b2982)